client/src/components/UserDropdown/style.scss

```diff
@@ -0,0 +1,136 @@
+@import '../styles/variables.scss';
+
+.button-menu {
+  display: inline-block;
+  padding: 5px;
+  font-family: Open Sans;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 600;
+  line-height: 24px;
+  outline: none;
+
+  @media screen and (max-width: 770px) {
+    font-size: 0;
+  }
+
+  .avatar {
+    display: inline-block;
+    width: 34px;
+    height: 34px;
+    vertical-align: middle;
+    cursor: pointer;
+    background: url('../../assets/img/user-avatar.svg') no-repeat;
+  }
+
+
+  .up-triangle {
+    display: inline-block;
+    width: 0;
+    height: 0;
+    margin-left: 5px;
+    border-color: $black  transparent transparent;
+    border-style: solid;
+    border-width: 7px 5px 0;
+
+    @media screen and (max-width: 770px) {
+      display: none;
+    }
+  }
+
+
+  .down-triangle {
+    display: inline-block;
+    width: 0;
+    height: 0;
+    margin-left: 5px;
+    border-color: transparent transparent $black;
+    border-style: solid;
+    border-width: 0 5px 7px;
+
+    @media screen and (max-width: 770px) {
+      display: none;
+    }
+  }
+}
+
+.null {
+  display: none;
+}
+
+.dropdown-content {
+  position: absolute;
+  z-index: 1;
+  min-width: 160px;
+  margin-top: 10px;
+  font-size: 14px;
+  text-align: left;
+  background-color: $white;
+  box-shadow: 0 8px 16px 0 $light-shadow;
+
+  .decor {
+    z-index: 2;
+    width: 0;
+    height: 0;
+    margin-top: -5px;
+    margin-left: 92%;
+    border-color: transparent transparent $white;
+    border-style: solid;
+    border-width: 0 5px 7px;
+
+    @media screen and (max-width: 770px) {
+      display: none;
+    }
+
+  }
+
+  .user-name {
+    padding: 10px 25px 0;
+    font-family: Open Sans;
+    font-size: 12px;
+    font-style: normal;
+    font-weight: 600;
+    line-height: 24px;
+   }
+
+  .email {
+    padding: 0 25px 10px;
+    font-family: Open Sans;
+    font-size: 12px;
+    font-style: normal;
+    font-weight: normal;
+    line-height: 19px;
+   }
+
+  a {
+    display: block;
+    padding: 10px 25px;
+    font-size: 14px;
+    line-height: 19px;
+    color: $black;
+    text-decoration: none;
+
+    &:hover {
+      width: 100%;
+      color: $crimson;
+      background-color: $light-alizarin;
+
+     }
+   }
+}
+
+.button-view {
```

Comment:
is it general style? why did u not to use wrapper for the component? pls ask Vitalii how he did it

Reply:
done